pkg/pool-manager/pool_test.go

```diff
@@ -413,5 +414,53 @@ var _ = Describe("Pool", func() {
 			Expect(newPod.Annotations[networksAnnotation]).To(Equal(afterAllocationAnnotation[networksAnnotation]))
 		})
 	})
+	Describe("Multus Network Annotations API Tests", func() {
+		It("should allocate requested mac-address with single ip-address request in network annotation", func() {
+			poolManager := createPoolManager("02:00:00:00:00:00", "02:00:00:00:00:01", &vmConfigMap)
```

Comment:
I know this follows the existing tests, but let's try to make our tests better today. Setup of the environment should be done in Context by BeforeEach. So in this case:

```go
Describe("... API Tests",... {

    Context("when pool is configured with two available addresses", ... {
        BeforeEach({
            // the pool setup
        })

        Context("and a Pod is created with single IP requested",...
            BeforeEach({
                // the VM setup
            })

            It("should successfullly ....", {
                // actual test
            })
    })
})

Reply:
ack